gtpv2/message/update-bearer-req.go

```diff
@@ -0,0 +1,415 @@
+// Copyright 2019-2021 go-gtp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package message
+
+import (
+	"github.com/wmnsk/go-gtp/gtpv2/ie"
+)
+
+// UpdateBearerRequest is a UpdateBearerRequest Header and its IEs above.
+type UpdateBearerRequest struct {
+	*Header
+	BearerContexts                *ie.IE
+	PTI                           *ie.IE
+	PCO                           *ie.IE
+	APNAMBR                       *ie.IE
+	ChangeReportingAction         *ie.IE
+	CSGInformationReportingAction *ie.IE
+	HeNBInformationReporting      *ie.IE
+	IndicationFlags               *ie.IE
+	PGWFQCSID                     *ie.IE
+	SGWFQCSID                     *ie.IE
+	PresenceReportingAction       *ie.IE
+	PGWNodeLoadControlInformation *ie.IE
+	PGWAPNLoadControlInformation  *ie.IE
+	SGWNodeLoadControlInformation *ie.IE
+	PGWOverloadControlInformation *ie.IE
+	SGWOverloadControlInformation *ie.IE
+	NBIFOMContainer               *ie.IE
+	PrivateExtension              *ie.IE
+	AdditionalIEs                 []*ie.IE
+}
+
+// NewUpdateBearerRequest creates a new UpdateBearerRequest.
+func NewUpdateBearerRequest(teid, seq uint32, ies ...*ie.IE) *UpdateBearerRequest {
+	c := &UpdateBearerRequest{
+		Header: NewHeader(
+			NewHeaderFlags(2, 0, 1),
+			MsgTypeUpdateBearerRequest, teid, seq, nil,
+		),
+	}
+
+	for _, i := range ies {
+		if i == nil {
+			continue
+		}
+		switch i.Type {
+		case ie.BearerContext:
```

Comment:
Could you double check by specks what IE are actually supported by UpdateBearer. Probably not all of these are needed

Reply:
All IE's are from specification one by one.